WORKSPACE

```diff
@@ -28,9 +28,9 @@ http_archive(
     patches = [
         "//third_party/io_grpc_grpc_java:054def3c63.patch",
         "//third_party/io_grpc_grpc_java:0959a846c8.patch",
+        "//third_party/io_grpc_grpc_java:1111913510.patch",
```

Comment:
Why changing the order of the patch file? The modification to the patch file made it compatible with `--incompatible_use_native_patch`, but changing the order of this patch file broke applying c63752789e.patch.